<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClipBoardQt</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="mTabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>440</width>
      <height>430</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="mTabClipBoard">
     <attribute name="title">
      <string>剪贴板</string>
     </attribute>
     <widget class="QLineEdit" name="mRegexpInput">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>330</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="mListWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>331</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>剪贴版列表</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>350</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>正则过滤器</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mFilterButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>370</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>过滤</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mDeleteButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>150</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mAddButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>110</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>插入</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mLoadDataButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>加载</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mOpenButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>310</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mCopyButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>230</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>拷贝</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mEditButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>190</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mSaveButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>70</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mPasteButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>270</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>粘贴</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mSettings">
     <attribute name="title">
      <string>设置</string>
     </attribute>
     <widget class="QLineEdit" name="mSettingInputSavePath">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>410</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>D:/#ClipBoardData</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>数据保存位置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mSettingSave">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>360</y>
        <width>75</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mSettingSave2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>360</y>
        <width>75</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>应用</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="mSettingListenSystemClipBoard">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>151</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>监听剪贴板</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="mSettingAutoStart">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>80</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>开机自启动</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="mSettingMinTray">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>161</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>启动时最小化到托盘</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="mSettingDeheavy">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>171</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>剪贴板去重</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
